### ISSUE TITLE
PANIC after bk configure

### ISSUE THREAD

**Issue by gramosbr** [buildkite/cli#35]
After successfully run `bk configure`, any other commands such `bk browse` errors with message:
```
🚨 %!!(MISSING)v(PANIC=runtime error: invalid memory address or nil pointer dereference)
```
Running on MacOs 10.14.3
Installed using homebrew
bk --version
0.3.0

**Comment by toolmantim** [comment]
Sorry you hit that error @gramosbr! Can you run it again with `--debug` and paste the output here?

**Comment by gramosbr** [comment]
Weirdly it only happens on the same terminal session I configured it.
Other sessions work. 
Not sure it's worth pursuing

```2019/02/13 14:57:32 [keyring] Considering backends: [file]
🚨 %!!(MISSING)v(PANIC=runtime error: invalid memory address or nil pointer dereference)
```

**Comment by TigerWolf** [comment]
I had this same problem. It happens when you are not within a git folder. 

In folder not controlled by git
```shell
# bk browse
🚨 %!!(MISSING)v(PANIC=runtime error: invalid memory address or nil pointer dereference)
```

In a folder with git but not one that is in Buildkite:
```shell
# bk browse
🚨 exit status 128
```

In a folder with git and in buildkite:
```shell
# bk browse
Enter passphrase to... [snip]
```

**Comment by nitrocode** [comment]
I'm experiencing the same issue after configuring and connecting with github successfully.

```
$ bk --version
1.1.0
$ bk configure
──── Ok! Let's get started with configuring bk 🚀


──── Configuring Buildkite GraphQL credentials

...

Securely stored GitHub token! 💪

Ok, you are good to go!
$ bk browse --debug
2020/09/03 15:01:00 [keyring] Considering backends: [pass file]
🚨 %!!(MISSING)v(PANIC=Error method: runtime error: invalid memory address or nil pointer dereference)
$ bk pipelines list
🚨 unexpected list
$ ls
pipeline.yml
```

**Comment by pda** [comment]
I suspect this is “solved” by removing keyring in #115 /  https://github.com/buildkite/cli/releases/tag/v2.0.0 😅
